README.md

```diff
@@ -523,6 +523,7 @@ XML
 * [Plump][71] - A lenient XML parser. [Artistic License 2.0][51].
 * [s-xml](http://cliki.net/S-XML) - A basic parser. [LLGPL][8].
 * [xmls](http://quickdocs.org/xmls/) - A small, simple, non-validating XML parser. [3-clause BSD][15].
+* [cl-feedparser](https://github.com/TBRSS/cl-feedparser) - A Common Lisp (RSS, Atom) feed parser. No license specified.
```

Comment:
The asd files states it's LLGPL.

Reply:
Woops, thanks ! fixed.

Reply:
Thanks for reviewing @rudolfochrist, and for fixing @vindarel!